### ISSUE TITLE
`sv_vote_yes_percentage` documentation is unclear

### ISSUE THREAD

**Issue by Robyt3** [ddnet/ddnet#7584]
The expected behavior would be that everyone on the server must vote yes for an option to pass when `sv_vote_yes_percentage 100` is set, but it fails immediately even if only a single player is on the server. It works with `sv_vote_yes_percentage 99`, so it looks like a rounding error somewhere.

**Comment by heinrich5991** [comment]
`sv_vote_yes_percentage 50` is the default and it should only pass when it's >50% I guess. So 100% makes it pass for >100%?

**Comment by 0xfaulty** [comment]
looks like could be fixed with replacing
`else if(No >= Total - Total / (100.0f / g_Config.m_SvVoteYesPercentage))`
with
`else if(No > 0 && No >= Total - Total / (100.0f / g_Config.m_SvVoteYesPercentage))`

**Comment by heinrich5991** [comment]
That special-cases 100%. All other values require "yes" votes to be above that percentage (not equal to).

**Comment by Robyt3** [comment]
If setting this to 100% doesn't make sense we could simply change the maximum value to 99% and maybe reword the description a bit.

**Comment by heinrich5991** [comment]
> If setting this to 100% doesn't make sense we could simply change the maximum value to 99% and maybe reword the description a bit.

Sounds good.
